src/dispatch/signal/scheduled.py

```diff
@@ -59,6 +59,7 @@ def process_signals(db_session: SessionLocal, project: Project):
         db_session.query(SignalInstance)
         .filter(SignalInstance.project_id == project.id)
         .filter(SignalInstance.filter_action == None)  # noqa
```

Comment:
`filter_action` gets set to `none` as string, but we seem to compare it to the `None` keyword, which SQLAlchemy converts it to `NULL`?

Reply:
Should we change the default value of `filter_action` in `SignalInstanceBase` to `none`?

Reply:
Something like `filter_action: SignalFilterAction = SignalFilterAction.none`?

Reply:
Then update this filter to compare with `SignalFilterAction.none`?

Reply:
Somehow it still works with the `None` here, but it's very confusing, I'll update it to work this way.

Reply:
Actually, we can't update this without making things complicated. Since we have set un filtered instances to `SignalFilterAction.none` here

```python
def filter_signal(*, db_session: Session, signal_instance: SignalInstance) -> bool:
    ...
    if not filtered:
        signal_instance.filter_action = SignalFilterAction.none
```

All of the existing instances in the database, without a case_id, will be re-processed.

It's probably smart to add a boundary on time created here know that I see this.